plop-load.md

```diff
@@ -1,76 +1,97 @@
 
 ## Examples
-*load via a path*
+
+_load via a path_
+
 ```javascript
-	// loads all 5 generators, no helpers, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js');
+// loads all 5 generators, no helpers, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js');
 ```
-*load via a path with a custom include config*
+
+_load via a path with a custom include config_
+
 ```javascript
-	// loads all helpers, no generators, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js', {}, { helpers: true });
+// loads all helpers, no generators, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js', {}, {helpers: true});
 ```
-*load via a path with a limited include*
+
+_load via a path with a limited include_
+
 ```javascript
-	// loads only the "js-header" helper, no generators, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js', {}, { helpers: ['js-header'] });
+// loads only the "js-header" helper, no generators, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js', {}, {helpers: ['js-header']});
 ```
-*load via a path with config object*
+
+_load via a path with config object_
+
 ```javascript
-	// the component.js module will receive the config object and do something
-	// example: it could use it to alter which templates it will use (es6)
-	//          and prefix all generator names with 'foo'
-	plop.load('./plopfiles/component.js', { es6: true, namePrefix: 'foo' });
+// the component.js module will receive the config object and do something
+// example: it could use it to alter which templates it will use (es6)
+//          and prefix all generator names with 'foo'
+plop.load('./plopfiles/component.js', {
+  es6: true,
+  namePrefix: 'foo'
+});
 ```
-*load via [npm module](https://www.npmjs.com/package/plop-pack-fancy-comments)*
+
+_load via [npm module](https://www.npmjs.com/package/plop-pack-fancy-comments)_
+
 > this module [configures a default include definition](https://github.com/amwmedia/plop-pack-fancy-comments/blob/master/index.js#L13)
 
 ```javascript
 	// loads all 3 helpers
 	plop.load('plop-pack-fancy-comments');
 };
 ```
-*load via npm module with a renaming include config*
+
+_load via npm module with a renaming include config_
+
 ```javascript
-	// loads only the header helper, no generators, actionTypes or partials (even if they exist)
-	// within the plopfile and templates, the "js-header" helper is referenced as "titleComment"
-	plop.load('plop-pack-fancy-comments', {}, { helpers: {'js-header': 'titleComment'} });
+// loads only the header helper, no generators, actionTypes or partials (even if they exist)
+// within the plopfile and templates, the "js-header" helper is referenced as "titleComment"
+plop.load(
+  'plop-pack-fancy-comments',
+  {},
+  {helpers: {'js-header': 'titleComment'}}
+);
```

Comment:
Softw wrap taked effect here. Can be changed at https://github.com/plopjs/plop/pull/237/files#diff-f2e285fb35cc9694564cb6d2af099ad2R15

Reply:
Thanks for pointing out. I'm fine with leaving as-is as long as that's what prettier wanted